eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -289,3 +289,52 @@ function Set-ChangeLogContent {
 
   Set-Content -Path $ChangeLogLocation -Value $changeLogContent
 }
+
+function Remove-EmptySections {
+  param (
+    [Parameter(Mandatory = $true)]
+    $ChangeLogEntry
+  )
+
+  $releaseContent = $ChangeLogEntry.ReleaseContent
+  $sectionsToRemove = @()
+
+  if ($releaseContent.Count -gt 0)
+  {
+    $parsedSections = $ChangeLogEntry.Sections
+    $sanitizedReleaseContent = New-Object System.Collections.ArrayList(,$releaseContent)
+  
+    foreach ($key in $parsedSections.Keys) 
+    {
+      if ([System.String]::IsNullOrWhiteSpace($parsedSections[$key]))
+      {
+        for ($i = 0; $i -lt $sanitizedReleaseContent.Count; $i++)
+        {
+          $line = $sanitizedReleaseContent[$i]
+          if ($line -eq "### $key")
```

Comment:
Lets refactor this match and the section regex above into a common helper function so that we don't end up making assumptions about how many header levels there are in multiple places. It will help when we start to think about how we fix issues like https://github.com/Azure/azure-sdk-for-java/issues/23908.

Reply:
I set the ithe regex a s a constant for the whole script.